arcsi/api/item.py

```diff
@@ -165,24 +167,37 @@ def add_item():
 
         # TODO get show cover img and set as fallback
         if request.files:
+            # Defend against possible duplicate files
```

Comment:
Isn't this check the same here and in edit? If yes, can you define it only once and use it where it is needed?

Reply:
Yes, that is long overdue. The reason I am hesitant to get at it or start a level-headed discussion because I want to do this by defining a separate media table put all this media server logic from the different object API's to their own. 

I would have added that "on the other hand -- it's OK not to touch it until this overhaul because it's trivial to update all the endpoints and apply the changes to their logic" but after tonight's miss on endpoint arguments I think it might be not that valid anymore and perhaps a good time to revisit this. 

Thanks for starting this train of thought!